modules/virt-creating-long-lived-account-and-token.adoc

```diff
@@ -0,0 +1,178 @@
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - k8s.cni.cncf.io
+    resources:
+      - network-attachment-definitions
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - storage.k8s.io
+    resources:
+      - storageclasses
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - kubevirt.io
+    resources:
+      - virtualmachines
+      - virtualmachines/finalizers
+      - virtualmachineinstancemigrations
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - kubevirt.io
+    resources:
+      - kubevirts
+      - virtualmachineinstances
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - cdi.kubevirt.io
+    resources:
+      - datavolumes
+      - datavolumes/finalizers
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+  - apiGroups:
+      - instancetype.kubevirt.io
+    resources:
+      - virtualmachineclusterpreferences
+      - virtualmachineclusterinstancetypes
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - instancetype.kubevirt.io
+    resources:
+      - virtualmachinepreferences
+      - virtualmachineinstancetypes
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+----
+. Run this command to create the service account:
++
+[source,terminal]
+----
+$ oc create serviceaccount <service_account_name> -n <service_account_namespace>
+----
+
+. Run this command to use a cluster role binding that links the service account to the cluster role:
++
+[source,terminal]
+----
+$ oc create clusterrolebinding <service_account_name> --clusterrole=<cluster_role_name>               --serviceaccount=<service_account_namespace>:<service_account_name>
+----
+
+. Create a secret to hold the token, as shown in the following example:
++
+[source,yaml]
+----
+cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <name of secret>
+  namespace: <namespace_you_want_sa>
+  annotations:
+    kubernetes.io/service-account.name: <sa_name>
+type: kubernetes.io/service-account-token
+EOF
+----
+
+. After the secret is populated (typically within a few seconds), run the following command:
++
+[source,terminal]
+----
+$ TOKEN_BASE64=$(oc get secret "<name_of_secret>" -n "<namespace_bound_to_service_account>" -o jsonpath='{.data.token}')
+  TOKEN=$(echo "$TOKEN_BASE64" | base64 --decode)
+  echo "$TOKEN"
+----
+
+. Copy the printed token.
+
+. In the {mtv-first} web console, when you create a provider and select *OpenShift Virtualization*, paste the token into the *Service account bearer token* field.
```

Comment:
Same comment as for https://github.com/openshift/openshift-docs/pull/101620/files#r2500074198